debian/debathena-pyhesiodfs.service

```diff
@@ -0,0 +1,20 @@
+# Author: Lizhou Sha <slz@mit.edu>
+# This file is part of the debathena-pyhesiodfs package.
+
+[Unit]
+Description=Hesiod automounter for Athena lockers
+Requires=openafs-client.service
+After=local-fs.target network.target openafs-client.service
+Before=umount.target
+
+[Service]
+Type=simple
+Environment="pyhesiodfs_dir=/mit"
+OOMScoreAdjust=-1000
+ExecStartPre=/bin/mkdir -p ${pyhesiodfs_dir}
+ExecStartPre=/bin/chown root:pyhesiodfs ${pyhesiodfs_dir}
+ExecStartPre=/bin/chmod 770 ${pyhesiodfs_dir}
+ExecStart=/usr/bin/pyhesiodfs -f ${pyhesiodfs_dir} -o nonempty
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
I think we should take out all declared relationships with `openafs-client`, on the theory that PyHesiodFS won’t actually fail with OpenAFS not running, and if you want non-broken symlinks, it’s your job to start both PyHesiodFS and OpenAFS.

`WantedBy=remote-fs.target` and even `Before=remote-fs.target` seems right, on the theory that the user might have an `@reboot` cron job that uses `/mit`. After looking at this further, it seems that Debian’s `cron.service` does not actually declare `After=remote-fs.target` like Fedora’s does (the latter turns out to be [scripts.mit.edu’s fault](https://bugzilla.redhat.com/show_bug.cgi?id=807029 scripts.mit.edu’s)). But the theory still seems logical…